packages/ui/src/components/va-badge/hooks/useFloatingPositionStyles.ts

```diff
@@ -36,19 +33,8 @@ export const useFloatingPosition = (
 ) => {
   if (!floating.value) { return {} }
 
-  let { position, align } = usePlacementAliases(props)
-
-  // TODO: remove `left` and `bottom` props in 1.6.0
-  if (props.left) {
-    position = 'left'
-    align = 'start'
-  }
-  if (props.bottom) {
-    position = 'bottom'
-    align = 'end'
-  }
-
-  const centerAlignment = computed(() => align === 'center' ? '-50%' : '0%')
+  const parsedPlacementAlias = usePlacementAliases(props)
```

Comment:
```suggestion
  const { position, align } = usePlacementAliases(props)
  
  poisition.value
  align.value
```

Would be nice to returns here not `Computed<{ position, align }>`, but `{ position: ComputedRef, align: ComputedRef }`